### PR TITLE
First-time Run without FxA

### PR DESCRIPTION
**Pull request by linuxwolf**
This implements the re-worked first-time run, using a traditional and "decoupled" master password flow.

All of the FxA client/server work is removed for now, but the basic interfacing still exists with the assumption that some form of it will happen in the future.

### CODE REVIEW

**Comment by linuxwolf** [comment]
@jimporter @m8ttyB this should be ready for review.

**Comment by jimporter** [comment]
@linuxwolf What's the long-term plan here? Are we just going to back this out when we get production telemetry?

**Comment by linuxwolf** [comment]
@jimporter 

1. FxA gets backed out before first alpha release.

2. FxA goes back in when we've got the client OAuth credentials, and the user experiments lead that way (as I understand it).

@sandysage @changecourse can comment better.

**Comment by m8ttyB** [comment]
QA approval: I can't find any faults with the workflow that are blocking a merge. I would like to re-test after we've merged all outstanding PRs and have generated a signed `.xpi` before going to Alpha.

**Comment by sandysage** [comment]
Correct. This next effort of UX is around exploring opportunities of leveraging FxA with the master password in a fairly tightly coupled interaction (much like we had previously): navigating the constraints, designing that experience, and pulling out what we need to do versus where we'll need support from the FxA team.

File: src/webextension/firstrun/components/app.js [hunk 1]
```diff
@@ -4,11 +4,21 @@
...
+import Welcome from "./welcome";
+import MasterPasswordSetup from "./master-password-setup";
+
+import styles from "./app.css";
 
 export default function App() {
   // Eventually, we'll have a feedback button up top here, and maybe some other
   // stuff.
-  return <Wizard pages={Pages}/>;
+  const imgSrc = browser.extension.getURL("/images/lockie_v1.png");
```
Comment (jimporter) [inline src/webextension/firstrun/components/app.js]: I feel like we can make Webpack import images and manage everything for us, but I haven't really looked into that yet...

File: src/webextension/firstrun/components/welcome.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import styles from "./welcome.css";
+
+export default function Welcome() {
+  return (
+    <section className={styles.welcome}>
+      <Localized id="welcome-title">
+        <h1>wELCOMe</h1>
+      </Localized>
+      <Localized id="welcome-intro">
+        <p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Sed posuere consectetur est at lobortis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam porta sem malesuada magna mollis euismod. Cras mattis consectetur purus sit amet fermentum.</p>
```
Comment (jimporter) [inline src/webextension/firstrun/components/welcome.js]: Wrap to 80 characters, please.

File: src/webextension/firstrun/components/welcome.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      </Localized>
+      <Localized id="welcome-feedback">
+        <p>Curabitur blandit tempus porttitor. Nulla vitae elit libero, a pharetra augue. Vestibulum id ligula porta felis euismod semper. Maecenas sed diam eget risus varius blandit sit amet non magna. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Maecenas sed diam eget risus varius blandit sit amet non magna.</p>
```
Comment (jimporter) [inline src/webextension/firstrun/components/welcome.js]: Ditto on the long line here.

File: src/webextension/firstrun/components/app.js (hunk 1 above)
Comment (linuxwolf) [reply src/webextension/firstrun/components/app.js]: I'd feel more confident about it if I knew what the `data:` URL breakpoint was, and how it interacts with `moz-extension://`.

**Comment by linuxwolf** [comment]
r=@jimporter IRL

**Pull request merged**